editor/voxel_debug.cpp

```diff
@@ -118,6 +118,7 @@ class DebugRendererItem {
 		// The reason is still unknown.
 		// It should be off anyways, but it's rather concerning.
 		_mesh_instance.set_cast_shadows_setting(VisualServer::SHADOW_CASTING_SETTING_OFF);
+		_mesh_instance.set_portal_mode(VisualServer::INSTANCE_PORTAL_MODE_GLOBAL);
```

Comment:
What is this class used for? I assumed this is a class to render collision shapes for static bodies but maybe I'm wrong?

Reply:
It is used for multiple things, most of which to highlight several chunking systems. It is not used for actual game stuff, as it names implies, it is for debugging.

Reply:
@Zylann If a portal system is active either in-game or in the editor we still want this to be visible, setting the portal mode to INSTANCE_PORTAL_MODE_GLOBAL will always make it visible, just like the terrain. Otherwise, if such a mesh is added to the server after a portal system is activated it will be initialised as INSTANCE_PORTAL_MODE_STATIC and not be visible (because STATIC instances need to be there before the room system is initialized and never move).

Reply:
If it is such a good default, I wonder why it isn't the default of newly created VisualInstances 🤔